### DISCUSSION TITLE
Caso de uso - Universidad Escuela Colombiana de Ingeniería

### DISCUSSION THREAD

**Discussion by rcfdtools** [rcfdtools/R.TeachingResearchGuide#2] [General]
https://github.com/rcfdtools/R.TeachingResearchGuide/blob/main/CaseUse.md

_Realice o publique aquí consultas, ideas, comentarios, preguntas y respuestas relacionadas con este documento wiki._

**Comment by juanrodace** [comment]
Después de la visita a la sala Laura y revisando algunos ejemplos de clases en video, con [MFL](https://github.com/mflatouche) y [AHOC](https://github.com/andresotalora92), sugerimos utilizar una presentación plana con fondo gris **[(HEX:#F8F8F8, R:248, G:248, B:248)](https://www.color-hex.com/color/f8f8f8)**. Igualmente, en la plantilla del video se puede utilizar un formato como el siguiente: https://www.youtube.com/watch?v=0eT5KqhntUs.

**Reply by juanrodace** [reply]
Adjunto plantilla. 
[Plantilla_PPTX_Videos.pptx](https://github.com/rcfdtools/R.TeachingResearchGuide/files/9262634/Plantilla_PPTX_Videos.pptx)

**Reply by rcfdtools** [reply]
Luego de la revisión de la plantilla preliminar enviada por @juanrodace y de realizar ajustes en los estilos del Slide Master para disponer de diseños para presentación con o sin expositor, se ha publicado la nueva versión de plantilla que contiene los siguientes estilos:

- Tema global de R.TeachingResearchGuide
- Diapositiva de bienvenida e introducción
- Título y objetos con expositor
- Título y objetos sin expositor
- Encabezado de sección con expositor
- Encabezado de sección sin expositor
- Dos objetos con expositor
- Dos objetos sin expositor
- Comparación con expositor
- Comparación sin expositor
- Solo el título con expositor
- Solo el título sin expositor
- En blanco
- Contenido con título sin expositor
- Imagen con título con expositor
- Imagen con título sin expositor

El nuevo color de fondo para lectura con relajamiento visual es R:240, G:240, B:240.
